dZoneManager/dZoneManager.cpp

```diff
@@ -27,19 +27,26 @@ void dZoneManager::Initialize(const LWOZONEID& zoneID) {
 	LOT zoneControlTemplate = 2365;
 
 	std::stringstream query;
-	auto result = CDClientDatabase::ExecuteQuery("SELECT zoneControlTemplate, ghostdistance_min, ghostdistance, PlayerLoseCoinsOnDeath FROM ZoneTable WHERE zoneID = " + std::to_string(zoneID.GetMapID()));
+	auto result = CDClientDatabase::ExecuteQuery("SELECT zoneControlTemplate, ghostdistance_min, ghostdistance FROM ZoneTable WHERE zoneID = " + std::to_string(zoneID.GetMapID()));
 
 	if (!result.eof()) {
 		zoneControlTemplate = result.getIntField("zoneControlTemplate", 2365);
```

Comment:
Why use CDZoneTableTable::Query only for acquiring the PlayerLoseCoinsOnDeath value? Why not use it for everything?

Reply:
Yea I did ask above if I should change both but I wanted to keep this PR on topic. I also noticed that whoever setup the constructor inside `CDZoneTableTable` also set the null value from the DB to be -1, but they're being stored in uints, so its causing a integer underflow. These would also need to be changed to either signed ints, or just change the deafult to 0.

Reply:
IIUC, the c/c++ specifications defines integer underflow on unsigned integers to be modulo width (i.e 32 here), so setting it to -1 is well defined and equal to `0xFFFFFFFF`

c.f. https://stackoverflow.com/a/2760612/4087068

Reply:
> IIUC, the c/c++ specifications defines integer underflow on unsigned integers to be modulo width (i.e 32 here), so setting it to -1 is well defined and equal to `0xFFFFFFFF`
> 
> c.f. https://stackoverflow.com/a/2760612/4087068

Oh thats good to know - I knew it existed just not the c/c++ specifics for how it was defined.